src/types/map.go

```diff
@@ -410,6 +410,11 @@ func (m Map) LazyArrayElemType() Map {
 
 	mm := make(map[string]struct{}, m.Len())
 	for typ := range m.m {
+		// TODO: https://github.com/VKCOM/noverify/issues/1227
+		if typ[0] >= WMax {
```

Comment:
```go
typ == "" || typ[0] >= WMax
```

And it's better to make it a function. And replace this pattern with this function :)